content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,314 @@
+`docker info --format '{{.Plugins.Log}}'` to check if the `journald` plugin is
+listed.
+- Network connectivity between Terraform Enterprise and the desired external
+destination(s) where logs should be forwarded to.
+
+## Configuration
+
+In order to use log forwarding, it must be enabled and configured. Once that is
+done, the Terraform Enterprise application will need to be restarted for the log
+forwarding configuration to take effect.
+
+### Enable Log Forwarding
+
+To enable log forwarding, the `log_forwarding_enabled` Terraform Enterprise
+application setting must be set to the value `1`.
+
+Enable log forwarding for a Standalone installation of Terraform Enterprise:
 
-```shell
-$ docker run --name="logspout" \
-  --volume=/var/run/docker.sock:/var/run/docker.sock \
-  gliderlabs/logspout \
-  syslog+tls://logs.mycompany.com:55555
+```
+replicatedctl app-config set log_forwarding_enabled --value 1
 ```
 
-The logspout container uses the Docker API internally to find other running containers and ingress their logs, then send them to `logs.mycompany.com` on port 55555 using syslog with TCP/TLS.
+Enable log forwarding for an Active/Active installation of Terraform Enterprise:
 
-~> **NOTE:** While docker has support for daemon-wide log drivers that can send all logs for all containers to various services,
-   Terraform Enterprise only supports having the Docker `log-driver` configured to either `json-file` or `journald`.
-   All other log drivers prevent the support bundle functionality from gathering logs, making it
-   impossible to provide product support. **DO NOT** change the log driver of an installation to anything other than `json-file` or `journald`.
+```
+tfe-admin app-config -k log_forwarding_enabled -v 1
+```
 
-## Audit Logs
+When enabled, the Terraform Enterprise application settings should show the
+following for the `log_forwarding_enabled` setting:
+
+```
+    ...
+    "log_forwarding_enabled": {
+        "value": "1"
+    },
+    ...
+```
+
+Log forwarding is disabled by default.
+
+### Configure External Destinations
+
+Once log forwarding is enabled, it must be configured to forward logs to one or
+more external destinations. To configure log forwarding, the
+`log_forwarding_config` Terraform Enterprise application setting must contain
+valid [Fluent Bit `[OUTPUT]` configuration](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit)
+specifying which external destination(s) to forward logs to. Since the Terraform
+Enterprise application settings are stored as JSON strings, it is recommended to
+create a `fluent-bit.conf` file with the valid Fluent Bit `[OUTPUT]`
+configuration and then configure the `log_forwarding_config` Terraform
+Enterprise application setting with the contents of that `fluent-bit.conf` file.
+That way, the configuration is stored in the Terraform Enterprise application
+settings exactly how it appears in the `fluent-bit.conf` file.
```

Comment:
```suggestion
The `log_forwarding_config` Terraform Enterprise application setting must contain valid [Fluent Bit `[OUTPUT]` configuration](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit) specifying external destination(s) where Terraform Enterprise should forward logs. Since the Terraform Enterprise application settings are stored as JSON strings, we recommend first creating a `fluent-bit.conf` file with the valid Fluent Bit `[OUTPUT]` configuration and then using that file to configure the `log_forwarding_config` application setting. This method ensures that the configuration is stored in the application settings exactly how it appears in the `fluent-bit.conf` file.
```